client/src/app/site/pages/meetings/modules/poll/services/vote-controller.service/vote-controller.service.ts

```diff
@@ -17,4 +17,8 @@ export class VoteControllerService extends BaseMeetingControllerService<ViewVote
     ) {
         super(controllerServiceCollector, Vote, repo);
     }
+
+    public async setHasVotedOnPoll(...viewPolls: ViewPoll[]): Promise<void> {
+        this.repo.setHasVotedOnPoll(...viewPolls);
```

Comment:
Please put the logic from the repo's function `setHasVotedOnPoll` in this function and make the repo's function `hasVotedFor` public.